cmd/controller-manager/app/controller_manager.go

```diff
@@ -342,3 +349,27 @@ func StartControllers(s *options.ControllerManagerServer,
 
 	select {}
 }
+
+var _ healthz.HealthzChecker = checkAPIAvailableResources{}
```

Comment:
This is a redundant check. It is done above in `healthz.InstallHandler(mux, healthz.PingHealthz, apiAvailableChecker)` code